internal/blueprint/blueprint.go

```diff
@@ -301,7 +301,8 @@ func Convert(bp Blueprint) iblueprint.Blueprint {
 		if fs := c.Filesystem; fs != nil {
 			ifs := make([]iblueprint.FilesystemCustomization, len(fs))
 			for idx := range fs {
-				ifs[idx] = iblueprint.FilesystemCustomization(fs[idx])
+				ifs[idx].Mountpoint = fs[idx].Mountpoint
+				ifs[idx].MinSize = fs[idx].MinSize
```

Comment:
nitpick: this should not be needed, since the struct didn't change in the end, right?